### PR TITLE
Javascript javascript3 week2

### PR DESCRIPTION
**Pull request by TatjanaOcedova**

### CODE REVIEW

File: javascript/javascript3/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    console.log(response);
+    const cards = await response.json();
+    return cards;
+  } catch (error) {
+    console.log(error);
+  }
+}
+setTimeout(() => {
+  deckOfCard();
+}, 3000);
```
Comment (Falafelqueen) [inline javascript/javascript3/week2/main.js:98]: Very good job!

File: javascript/javascript3/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        const latitude = position.coords.latitude;
+        const longitude = position.coords.longitude;
+        position = { latitude, longitude };
+        resolve(position);
+      });
+    } else {
+      reject;
+    }
+  });
+}
```
Comment (Falafelqueen) [inline javascript/javascript3/week2/main.js:53]: 👏

File: javascript/javascript3/week2/promise-visual/main.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }),
+    moveElement(circles[2], {
+      x: 400 - parseInt(circles[2].style.left),
+      y: 20 - parseInt(circles[2].style.top),
+    }),
+  ])
+    .then(() => {
+      console.log("All circles are moved");
+    })
+    .catch((error) => console.error(err));
```
Comment (Falafelqueen) [inline javascript/javascript3/week2/promise-visual/main.js]: Great job ! Only a tiny thing in here, the error and err you are logging out should be the same thing :-)

**Review by Falafelqueen — APPROVED** [review]
Great job ! Promises are not easy at all, and you are nailing it.🦾

File: javascript/javascript3/week2/main.js (hunk 1 above)
Comment (TatjanaOcedova) [reply javascript/javascript3/week2/main.js:98]: Thank you 😊

File: javascript/javascript3/week2/promise-visual/main.js (hunk 3 above)
Comment (TatjanaOcedova) [reply javascript/javascript3/week2/promise-visual/main.js]: Thank you 😊 I will edit it. I use a little help how to call circle elements it was a little tricky for me

**Pull request merged**